ContactManager/src/com/contacts/manager/ContactManager.java

```diff
@@ -0,0 +1,77 @@
+package com.contacts.manager;
+
+import com.contacts.model.Contact;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class ContactManager {
+
+    private static final ContactManager myClass = new ContactManager();
+    private static List<Contact> contacts = new ArrayList<Contact>();
+
+    private ContactManager() {
+
+    }
+
+    public static ContactManager getInstance() {
```

Comment:
@cesar1414 Singleton pattern would not be a great solution when we try link a the Contact Manager per user.